### PR TITLE
Refactor claim rewards to not use the backend

### PR DESCRIPTION
**Pull request by julia-script**
# 🧙 Description
I missed this part, it was still using the go backend

Ticket #

## ✅ Checklist

- [ ] Acceptance Criteria described in the ticket are met
- [ ] Texts and strings are in the related i18n file
- [ ] Utilities have unit tests
- [ ] User stories and functionalities have integration or e2e tests
- [ ] If adding a new token or network
  - [ ] Registry package is updated
  - [ ] Asset images are added
- [ ] Related packages.json are upgraded
- [ ] Changelog is updated

### CODE REVIEW

File: packages/helpers/src/clients/get-pub-url.ts [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
+        VERCEL_URL ||
+        NEXT_PUBLIC_VERCEL_URL ||
         `http://localhost:${process.env.PORT}`
       );
     }
     return (
-      process.env.VERCEL_URL ||
-      process.env.NEXT_PUBLIC_VERCEL_URL ||
-      "https://staging-app.evmos.org"
+      VERCEL_URL || NEXT_PUBLIC_VERCEL_URL || "https://staging-app.evmos.org"
```
Comment (miliPaoletti) [inline packages/helpers/src/clients/get-pub-url.ts:22]: shouldn't this be https://staging-store.evmos.org ?

File: packages/helpers/src/evmos-info.ts [hunk 2]
```diff
@@ -13,16 +13,18 @@ export const evmosmainnet = {
     },
   },
   rpcUrls: {
-    default: { http: [`${getPubUrl()}/api/eth-json-rpc/evmos`] },
-    public: { http: [`${getPubUrl()}/api/eth-json-rpc/evmos`] },
+    default: { http: ["https://proxy.evmos.org/cosmos"] },
```
Comment (miliPaoletti) [inline packages/helpers/src/evmos-info.ts:16]: Screenshot 2024-06-06 at 10 11 00

heeeey Julia! Any reason why you change it ? With that update if you try to liquid stake or swap it doesn't work. 
I remember that I tried changing to `/web3` and it worked, but not sure why you changed it

**Review by miliPaoletti — APPROVED** [review]

File: packages/helpers/src/evmos-info.ts (hunk 2 above)
Comment (julia-script) [reply packages/helpers/src/evmos-info.ts:16]: oh, I think I f. up, thanks for catching this :D

File: packages/constants/src/env-variables.ts [hunk 3]
```diff
@@ -67,3 +67,9 @@ export const EVMOS_LANDING_PAGE_NOTION_ID =
   "96437b2af5ec48bc9772a8304c918466";
 
 export const CHAIN_REGISTRY_REF = process.env.CHAIN_REGISTRY_REF || "main";
+
+export const CLAIM_REWARDS_THRESHOLD = BigInt(
+  process.env.NEXT_PUBLIC_CLAIM_REWARDS_THRESHOLD || "5000000000000000",
```
[the remark this answers is not in the document]
Comment (julia-script) [reply packages/constants/src/env-variables.ts:72]: it's only necessary if you want anything different from the default :)

**Pull request merged**